OutSystems/outsystems-datagrid
# ROU-4186: New block for the click event on the grid

This PR is for add a new ClickEvent block.

### What was done
* Created a new ClickEvent feature
* Created a new GridOnCellClickEvent event
* Edited the FeatureBuilder fo include the clickEvent feature

### Test Steps
1. Add a the ClickEvent block to the page after the Grid
2. Add a handler
3. Click in a cell on the Grid
4. Check if the handler receive column, row and data information related to the clicked cell.

### Checklist
